Update Astro Alerts

lzdanski opened the pull request:
Resolves https://github.com/astronomer/docs/issues/2226
Resolves https://github.com/astronomer/docs/issues/2685

To-do
- [x] Test the manual failures
- [x] add screen shots from issues
- [x] Add runtime version requirement
- [x] Add testing steps

jwitz commented on astro/alerts.md:
```diff
@@ -116,4 +146,22 @@ In the Cloud UI, you can enable alerts from the **Workspace Settings** page.
+
+3. Trigger a DAG run.
+
+4. Manually mark your DAG's state as **Failed** by clicking the DAG run in the Grid view, then clicking **Mark state as...** and choose **Failed**.
```
```suggestion
4. Manually mark your DAG's state as **Failed** by clicking the DAG run in the Grid view, then clicking **Mark state as...** > **Failed**.
```

jwitz reviewed (changes requested):
New content looks good! Left a question about the screenshots, I'm a little nervous when it comes to documenting third party processes through images like that

manmeetkaur commented on astro/alerts.md:
```diff
@@ -44,10 +50,16 @@ To set up alerts in Slack, you need to create a Slack app in your Slack workspac
 
 5. On the **Incoming webhooks** page, click to turn on **Activate Incoming Webhooks**.
 
+  ![Activate an incoming webhook](/img/docs/slack_alerts_1.5.png)
```
That is a good point! 👍

manmeetkaur reviewed (commented):
just 1 small question, otherwise LGTM! 👍

lzdanski commented:
Hey @jwitz - I know this had gotten a little stale but based on review:

- Renamed screenshots to not refer to the steps they appear in
- Added screenshots for updated UI that includes email alerts
- Removed most of the 3rd party screenshots (need to leave the pagerduty one, it was really confusing for us to find the "integrations" even with a direct link)
- Added a black border to all images

jwitz commented on astro/alerts.md:
```diff
@@ -141,4 +149,22 @@ In the Cloud UI, you can enable alerts from the **Workspace Settings** page.
\ No newline at end of file
+8. Click **Create alert**.
+
+## Step 3 (Optional): Test your DAG run failure alert
```
```suggestion
## Step 3: (Optional) Test your DAG run failure alert
```
This seems slightly more consistent with our optional styling in steps

jwitz commented on astro/alerts.md:
```diff
@@ -141,4 +149,22 @@ In the Cloud UI, you can enable alerts from the **Workspace Settings** page.
+
+## Step 3 (Optional): Test your DAG run failure alert
+
+Astro alerts work whether your DAG run is manual or scheduled. So, you can easily test your configures Astro alert by failing your DAG manually.
```
```suggestion
Astro alerts work whether your DAG run is manual or scheduled, so you can test your configured Astro alerts by failing your DAG manually.
```

jwitz commented on astro/alerts.md:
```diff
@@ -141,4 +149,22 @@ In the Cloud UI, you can enable alerts from the **Workspace Settings** page.
+
+Astro alerts work whether your DAG run is manual or scheduled. So, you can easily test your configures Astro alert by failing your DAG manually.
+
+1.  In the Cloud UI, open the Deployment where you configured your alert and click **Open Airflow**.
```
```suggestion
1. In the Cloud UI, open the Deployment where you configured your alert and click **Open Airflow**.
```

jwitz commented on astro/alerts.md:
```diff
@@ -141,4 +149,22 @@ In the Cloud UI, you can enable alerts from the **Workspace Settings** page.
+
+1.  In the Cloud UI, open the Deployment where you configured your alert and click **Open Airflow**.
+
+2. Choose the DAG that has your alert configured.
```
I think we want to start driving usage of the DAGs tab in the Cloud UI, which is basically a reskinned Airflow UI that's more integrated with Astro features. These steps look great, but can you tweak the wording and screenshots to use the DAGs page workflow instead?

![Screenshot 2023-08-16 at 11 55 48 AM](https://github.com/astronomer/docs/assets/74574233/d0ac3f92-3814-484b-a947-05682274797f)

jwitz commented on static/img/docs/astro_alerts_slack.png:
Being v picky here, but we could trim the bottom of these screenshots so that we're just showing the required config area

jwitz reviewed (changes requested):
Looks great @lzdanski ! Love the new test section. Only requesting changes because I'd like to see that setup use the Cloud UI **DAGs** page if possible.

jwitz reviewed (approved):
LGTM! Lmk if you need help with the Snagit borders
